### PR TITLE
Mention official introduction package install in QuickInstall

### PR DESCRIPTION
**Pull request by jonaseberle**
The TYPO3 official introduction package is a great possibility for new users to look around and experience TYPO3. It would market TYPO3 a lot better than the "empty start page". I suggest to add it to the QuickInstall manual. in my opinion this will add value for newcomers. Unfortunatley with composer this possiblity has become a lot less obvious (See also https://forge.typo3.org/issues/88940) why IMHO we need to mention it here.

### CODE REVIEW

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 1]
```diff
@@ -79,7 +79,28 @@ The Install Tool
       :class: with-shadow
 
       Install Tool in 1-2-3 mode, fifth step.
-
+      
+   **Introduction distribution**
```
Comment (tomwarwick) [inline Documentation/QuickInstall/TheInstallTool/Index.rst]: **Introduction Package**

    If you are using TYPO3 for the first time you may want to
    see a working example of the CMS before you start work on
    your own project.
    The `Official Introduction Package
    <https://extensions.typo3.org/extension/introduction/>`__ showcases many of TYPO3s
    capabilities and gives you the ability to try them first hand.
    The Introduction Package utilizes the `bootstrap_package extension
    <https://extensions.typo3.org/extension/bootstrap_package/>`__
    to generate multiple responsive HTML templates that you can select and try out.
    It also features examples of the different kinds page content that you typically see
    on a website. Such as paragraphs of text, images, tables and navigation menus.

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 2]
```diff
@@ -79,7 +79,28 @@ The Install Tool
...
+   If you are new to TYPO3 you might want to start with example pages and 
+   content added. The `official Introduction Package 
+   <https://extensions.typo3.org/extension/introduction/>`__ showcases TYPO3s 
+   capabilities and gives you a head start to play around with a functioning 
+   site. It uses the `extension bootstrap_package
+   <https://extensions.typo3.org/extension/bootstrap_package/>`__ which 
+   uses the Bootstrap CSS framework and adds custom content elements to make
+   use of it within TYPO3. 
+   
+   #. **If you installed TYPO3 with composer**
```
Comment (tomwarwick) [inline Documentation/QuickInstall/TheInstallTool/Index.rst]: You will need to `add the composer package and enable the extension in the backend of the CMS
       <https://docs.typo3.org/typo3cms/extensions/introduction/stable/Installation.html>`__.

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 3]
```diff
@@ -79,7 +79,28 @@ The Install Tool
...
+      
+      You have to `add a composer package and enable the extension
+      <https://docs.typo3.org/typo3cms/extensions/introduction/stable/Installation.html>`__.
+
+   #. **If you installed TYPO3 without composer**
```
Comment (tomwarwick) [inline Documentation/QuickInstall/TheInstallTool/Index.rst]: You can select "Yes, download the list of distributions" when you finish installing the CMS.
       Then, when you log in to the backend for the first time, you will be presented with
       a list of available packages including the "Official Introduction Package".

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 4]
```diff
@@ -79,7 +79,36 @@ The Install Tool
       :class: with-shadow
 
       Install Tool in 1-2-3 mode, fifth step.
-
+
```
Comment (sypets) [inline Documentation/QuickInstall/TheInstallTool/Index.rst:82]: I agree, that we should mention introduction package in Install & Upgrade guide, the question is just where and how.

* Currently there already is a section: In the "In depth Installation" chapter: https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/In-depth/Distributions/Index.html#introduction-package We should be careful not to create too much duplicate information
* should the Introduction package really be in the "Install Tool" page? (when installing it without Composer, it makes sense. When installing it with Composer it probably does not). In that case we should probably add it to this page: https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/QuickInstall/Composer/Index.html. But it already exists in the In depth section: https://docs.typo3.org/m/typo3/guide-installation/master/en-us/In-depth/Distributions/Index.html#installing-distributions-with-composer

**Comment by sypets** [comment]
I could not push my changes because I rebased, So I created a new PR #163 based on your commits. 

It creates a new page for installing the introduction package in "Quick install". Also it adds some screenshots and makes some more minor changes. Closing here.

**Pull request closed without merging**